source/includes/_payouts.md

```diff
@@ -726,7 +726,7 @@ Field                          | Description                                   |
 ---------                      | -------                                       | -----------
 targetAccount                  | Recipient account id. You can create multiple transfers to same recipient account.   | Integer
 quote                          | Quote id. You can only create one transfer per one quote. <br/>You cannot use same quote ID to create multiple transfers. | Integer
-customerTransactionId     | This is required to perform idempotency check to avoid duplicate transfers in case of network failures or timeouts.                          | UUID
+customerTransactionId     | This is required to perform idempotency check to avoid duplicate transfers in case of network failures or timeouts.                          | Text
```

Comment:
In the case of `customerTransactionId` I believe a valid UUID is required: you will get errors when trying to use other text values. I think this is a case where we must document UUID until the system accepts other types of value.